tests/apidoc_presence_test.go

```diff
@@ -0,0 +1,26 @@
+package tests
```

Comment:
Please add a blacklist in for Dendrite as it does not implement presence yet. E.g https://github.com/matrix-org/complement/blob/33f712f8b1c138210ee9514c32f383f10937bb7a/tests/user_query_keys_test.go#L1 - this comment is read by Go, and it has to be the first line in the file. Replace `synapse` with `dendrite`.

Reply:
You will also need to add this tag to the Github Actions file - https://github.com/matrix-org/complement/blob/09ac587f5a1c96f056b8723fc1316d1e816c228f/.github/workflows/ci.yaml#L29 - add in `dendrite_blacklist`.

Doing this should make CI pass.